echopype/echodata/echodata.py

```diff
@@ -403,12 +403,29 @@ def update_platform(
             extra_platform_data = extra_platform_data.drop_vars(trajectory_var)
             extra_platform_data = extra_platform_data.swap_dims({"obs": time_dim})
 
-        # TODO: Expand the clipping range by one (or 2?) indices
+        # clip incoming time to 1 less than min of EchoData.beam["ping_time"] and
+        #   1 greater than max of EchoData.beam["ping_time"]
+        # account for unsorted time
+        sorted_external_time = extra_platform_data[time_dim].data
+        sorted_external_time.sort()
```

Comment:
Could you move .sort to the top of that you have `sorted_external_time = extra_platform_data[time_dim].data.sort()` ?

Reply:
The sort is in place, .sort() doesn't return anything

Reply:
Ah ok, somehow I missed that. Thanks!